src/test_fixture/input/sharing.rs

```diff
@@ -354,3 +354,33 @@ where
         }
     }
 }
+
+impl<TS, BK, TV> IntoShares<OprfReport<TS, BK, TV>> for TestRawDataRecord
+where
+    TS: GaloisField + IntoShares<Replicated<TS>>,
+    BK: GaloisField + IntoShares<Replicated<BK>>,
+    TV: GaloisField + IntoShares<Replicated<TV>>,
+{
+    fn share_with<R: Rng>(self, rng: &mut R) -> [OprfReport<TS, BK, TV>; 3] {
+        let event_type = if self.is_trigger_report {
+            EventType::Trigger
+        } else {
+            EventType::Source
+        };
+        let timestamp = TS::truncate_from(self.timestamp).share_with(rng);
+        let breakdown_key = BK::truncate_from(self.breakdown_key).share_with(rng);
+        let trigger_value = TV::truncate_from(self.trigger_value).share_with(rng);
```

Comment:
Should probably not use `truncate_from` as this will hide errors (like we encountered today!) we should fail spectatularly.